docs/pages/kubernetes-access/getting-started.mdx

```diff
@@ -178,13 +178,13 @@ Create the role and add a user:
 
 ```bash
 # To create a local user, we are going to run Teleport's admin tool tctl from the pod.
-$ POD=$(kubectl get po -l app=teleport-cluster -o jsonpath='{.items[0].metadata.name}')
```

Comment:
this is not a typo, this should work

Reply:
TIL Kubernetes supports `po` as an [abbreviation](https://kubernetes.io/docs/reference/kubectl/overview/#syntax) of `pod` I'll double check that is wasn't an issue with something else

Reply:
I've reverted this change.